packages/client/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "vergunningcheck-client",
+  "name": "client",
```

Comment:
There was an issue with the `-` in the package.json files using Lerna. Removed all the `vergunningcheck-`. Fortunately graphql still works.

Reply:
Maybe that's why Netlify is not working as well